### DISCUSSION TITLE
Call to undefined method $this->pull()

### DISCUSSION THREAD

**Discussion by Akhmami** [livewire/livewire#8518] [1. Help] [answered]
I'm using livewire v3.5 I get an error on the form object where the pull() method is used.

```php
// Call to undefined method App\Livewire\Forms\PhaseForm::pull()

public function store(Period $period)
    {
        $this->validate();

        $period->phases()->create(
            $this->pull()
        );
    }
```

**Comment by joshhanley** [comment]
@Akhmami I'm pretty sure pull only exists on the component, not the form object

**Reply by Akhmami** [reply]
The documentation works on the form object.

**Comment by PerryvanderMeer** [comment, marked as answer]
It's available since #8481, but it's not tagged yet, so it will be in the next release.
